### PR TITLE
Awesome Books Project Foundation

### PR DESCRIPTION
**Pull request by DevKaranJ**
This pull request lays the groundwork for the Awesome Books project, setting up the initial project structure, integrating HTML, CSS, and JavaScript components, and configuring linters to maintain code quality and consistency.

With this initial setup complete, the project is now ready for the implementation of features outlined in the project requirements. Future PRs will build upon this foundation, enhancing the user interface, adding book management capabilities, and persisting data using local storage.

Your feedback and suggestions on this initial setup are greatly appreciated as we embark on the journey of creating the Awesome Books website.

Thank you!

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @DevKaranJ and @richie1988,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly provide a live demo link or remove that section if you have no live link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,164 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone-project-Music-Festival-Website] <a name="about-project"></a>
```
Comment (muneebulrehman) [inline README.md]: - Optional
It seems like you are using the title form your previous project. I would recommend updating it. Remember, having a professional repo is super important.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    const removeButton = document.createElement('button');
+    removeButton.textContent = 'Remove';
+    removeButton.addEventListener('click', () => {
+      removeBook(index);
+    });
+
+    li.appendChild(removeButton);
+    bookList.appendChild(li);
+  });
+}
```
Comment (muneebulrehman) [inline script.js:33]: As per the project requirements, when I reload the page, the existing books in the local storage should be populated on the UI. However, that is not the case in your application. Only when I add a book after reloading, do the existing books in local storage appear on the UI. Kindly fix this issue. You can read about `window.onload` event which can help you in this case.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good job adding and removing the books.
- Data stored in local storage.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shahadat3669 — APPROVED** [review]
Hi @DevKaranJ @richie1988 , 

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Data is preserved in the [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md).
✅ Has a function that adds a new book to the collection.
✅ has a function that removes a book from the collection.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**